### DISCUSSION TITLE
After upgrading my Taranis QX 7 to Edge TX, my models are all gone.  They are on the SD card but were never converted.

### DISCUSSION THREAD

**Discussion by MisterTee8227** [EdgeTX/edgetx#5548] [Q&A]
As the subject says, I upgraded my radio and my models never converted.  Is there a manual conversion process I can run through.  I was going to try try to factory reset it but I'm not sure how.  Aside from the models missing everything else looks fine.  Can someone help me get my models back?

**Comment by pfeerick** [comment]
Some important details are missing here: (1) upgraded from what (and
version of what) (2) upgraded to what version? (As if you are trying to
upgrade from OpenTX), you need to run EdgeTX 2.8.x for that conversion to
happen. (3) Can you use Companion? As that can read OTX files and do the
necessary conversion. (4) This probably should have been 1, but if this is
a (Q)X7, since settings are stored in EEPROM, not on SD... er... What was
your upgrade process?

**Reply by MisterTee8227** [reply]
Sorry.  I'm new at this.

I upgraded from OpenTX (not sure what version it was) to Edge TX 2.10.4 (the newest version as I downloaded it yesterday).  

I did download the companion but I'm not sure how to use it.  When I launch it now, I just get a virtual representation of the remote on screen.  I don't see any options for fixing models or converting them.

I backed up the current SD card to a file on my desktop.  I then wiped the card and put the sd contents from EdgeTX sd card zip onto my sdcard.  I downloaded the QX 7 firmware and flashed using the radio itself to flash the firmware off the sd card.  After the flashing, I rebooted and was met with a wizard to calibrate my sticks.  Once that was done, I was back to the radio home screen and it showed Model 1.  Hitting the button on the remote to pick the model shows only model 1 and all my previous models are missing.  I still have the bin files so I'm hoping there is a simple conversion step I can run after the fact.

**Reply by MisterTee8227** [reply]
forgot to mention that I copied the LUA scripts and models back to the SD card from my backup of the original SD card.  Prior to flashing per the instructions.
